R/ComputeShifts.R

```diff
@@ -0,0 +1,101 @@
+#' Adds leads or lags of a variable as new columns in a panel dataset
+#'
+#' @description `ComputeShifts` adds rows to a panel dataset with leads or lags of a variable.
+#'
+#' @param df Data frame that will be modified.
+#' @param idvar Character indicating column of optional grouping variable.
+#' @param timevar Character indicating column of time periods.
+#' @param shiftvar Character indicating column of variable that will be shifted.
+#' @param shiftvalues Numeric vector specifying the leads/lags to be computed. For example, '-1:1' will compute the lag, current, and lead values.
+#' @param timevar_holes Logical indicating whether the panel contains missing values in the time variable.
```

Comment:
Same question as above re: gaps vs. missing values.

Reply:
Same response as in https://github.com/JMSLab/eventstudyr/pull/38/files#r1189250252. Resolving this thread for simplicity.